spec/term/1.0.md

```diff
@@ -0,0 +1,204 @@
+<!-- draft -->
+# `vt6/term1.0` - Universal terminal capabilities and input/output handling
+
+The canonical URL for this document is <https://vt6.io/std/term/1.0/>.
+
+**This is a non-normative draft.**
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+This document uses the predefined property types from [section 2.4 of `vt6/core1.0`](https://vt6.io/std/core/1.0/#section-2-4).
+
+## 1. Scope
+
+This module defines how a terminal works, how clients' standard output is reflected on the terminal, and how user input is presented on clients' standard input.
+Besides [`vt6/core`](https://vt6.io/std/core/), this is the only module that we expect to be available in **every** VT6 server implementation across the entire feature spectrum, from graphical terminal emulators all the way down to continuous stationery printers.
```

Comment:
term module: Can we safely allow vt6 client implementers to assume the term module to always be present? Or should we advise them to always `want` it?